tests/data/vmware-sdwan.json

```diff
@@ -0,0 +1,2571 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.8072.3.2.10",
+                    "sysDescr": "VeloCloud EDGE610",
+                    "sysContact": "<private>",
+                    "version": null,
```

Comment:
No version number or build number or sth to populate this with?

Reply:
No, they don't expose the version of the software via snmp..

Reply:
No worries